README.md

```diff
@@ -38,4 +38,27 @@ export const getAbility = async () => {
 }
 ```
 
+if you want to use the POST Bulk endpoint, you can use the following code:
+```javascript
+import { Ability } from '@casl/ability';
+import { Permit, permitState } from 'permit-fe-sdk';
+export const getAbility = async () => {
+    const permit = Permit({loggedInUser: "odedbd@gmail.com", backendUrl: "http://localhost:4000/"});
+    await permit.loadLocalStateBulk([{ action: "create", resource: "file" }, { action: "update", resource: "file" }, { action: "delete", resource: "file" }, { action: "read", resource: "file" }]);
+    const caslConfig = permitState.getCaslJson();
+    return caslConfig && caslConfig.length? new Ability(caslConfig) : undefined ;
+}
+```
+
+In the above request, you also have the option to pass in resource_attributes that are needed for permissions check.
```

Comment:
Please add: Attributes are used solely for ABAC permission modeling. If using RBAC or ReBAC, leave it empty.